.github/workflows/apidiff.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       id: changed_dirs
       # Ignore changes to the internal and root directories.
       run: |
-        dirs=$(git diff-tree --no-commit-id --name-only --diff-filter=DMR -r ${{ steps.main.outputs.hash }}..HEAD | xargs -r -L1 dirname | uniq | grep -v -e 'internal' -e '\.' | cat)
+        dirs=$(git diff-tree --no-commit-id --name-only --diff-filter=DMR -r ${{ steps.main.outputs.hash }}..HEAD | grep -v -e 'go\.[mod|sum]' | xargs -r -L1 dirname | uniq | grep -v -e 'internal' -e '\.' | cat)
```

Comment:
Will this still work for modules like pubsub or storage that have sources at the module root?